tests/ert_tests/ert3/console/integration/test_cli.py

```diff
@@ -647,6 +644,24 @@ def test_cli_local_test_run_specific_realization(tmpdir):
                 ):
                     ert3.console._console._main()
 
+            with pytest.raises(
+                ert.exceptions.ConfigValidationError,
+                match="Realization out of ensemble bounds",
+            ):
+                with patch.object(
+                    sys,
+                    "argv",
+                    [
+                        "ert3",
+                        "run",
+                        "evaluation",
+                        "--local-test-run",
+                        "--realization",
+                        "200000",
```

Comment:
This is just nitpicking, but should `200000` be a variable than is then assert (being less than) with the number of realizations?